src/Functions/alphaTokens.cpp

```diff
@@ -99,7 +99,31 @@ using FunctionSplitByAlpha = FunctionTokens<SplitByAlphaImpl>;
 
 REGISTER_FUNCTION(SplitByAlpha)
 {
-    factory.registerFunction<FunctionSplitByAlpha>();
+    FunctionDocumentation::Description description = R"(
+Selects substrings of consecutive bytes from the ranges `a-z` and `A-Z` and returns an array of the selected substrings.
+)";
+    FunctionDocumentation::Syntax syntax = "alphaTokens(s[, max_substrings])";
+    FunctionDocumentation::Arguments arguments = {
+        {"s", "The string to split.", {"String"}},
+        {"[max_substrings]", "Optional. When `max_substrings > 0`, the returned substrings will be no more than `max_substrings`, otherwise the function will return as many substrings as possible.", {"Int64"}}
```

Comment:
```suggestion
        {"[max_substrings]", "Optional. When `max_substrings > 0`, the number of returned substrings will be no more than `max_substrings`, otherwise the function will return as many substrings as possible.", {"Int64"}}
```